lib/stringToCharacters.js

```diff
@@ -0,0 +1,9 @@
+function stringToCharacters(string) {
```

Comment:
Nice solution! You can also do this in one line!
```javascript
function stringToCharacters(string){
    return string.split('');
}
console.log(stringToCharacters('abschfoeh12'));
```
The `.split()` function is really cool and it splits a string by the parameter you give it. For example `'cat'.split('a')` would split by `a` and give you the array `['c', 't']`. In the code above, splitting by `''` (empty string, not space) gives an array with all characters, because there is an empty string between each character of the string.